artio-core/src/main/java/uk/co/real_logic/artio/session/Session.java

```diff
@@ -49,17 +55,44 @@
 import static uk.co.real_logic.artio.LogTag.FIX_MESSAGE;
 import static uk.co.real_logic.artio.builder.Validation.CODEC_VALIDATION_DISABLED;
 import static uk.co.real_logic.artio.builder.Validation.CODEC_VALIDATION_ENABLED;
-import static uk.co.real_logic.artio.dictionary.SessionConstants.*;
+import static uk.co.real_logic.artio.dictionary.SessionConstants.HEARTBEAT_MESSAGE_TYPE_CHARS;
```

Comment:
Does this PR need to have so much import statement churn in it?

Reply:
unfortunately that's my configuration of optimize imports, I'll revert this